rai/results_test.go

```diff
@@ -249,10 +249,11 @@ var primitiveTypeTests = []execTest{
 	},
 	{
 		query: `def output = auto_number[1]`,
-		mdata: mdata("0.arrow", sig("output", Int64Type,
-			vtype("rel:base:AutoNumber", Int64Type))),
+		// TODO (dba) AutoNumber was incorrectly tagged as having an `Int64Type`. Until the change in the engine is in prod, we need to keep it as `nil`.
+		// mdata: mdata("0.arrow", sig("output", Int64Type,
```

Comment:
@bachdavi is the comment still relevant here, since you ended up changing it to uint64 already?